source/features/prevent-pr-commit-link-loss.tsx

```diff
@@ -9,6 +9,10 @@ import * as textFieldEdit from 'text-field-edit';
 import features from '.';
 import {prCommitUrlRegex, preventPrCommitLinkLoss} from '../github-helpers';
 
+function toggleAllowEditsVisibility({visible}: {visible: boolean}) {
+	select<HTMLInputElement>('[name="collab_privs"]')?.closest('.float-left')?.classList.toggle('d-none', !visible);
```

Comment:
I could have used `#new_pull_request .float-left` but I felt it was not specific enough =.

Reply:
Why do this at all?

Reply:
Before
![image](https://user-images.githubusercontent.com/16872793/85937645-97a4ed80-b8d3-11ea-9e91-5a20d1fce446.png)

After

![image](https://user-images.githubusercontent.com/16872793/85937650-9b387480-b8d3-11ea-83ec-303dbf24b7cf.png)

Reply:
Can the conflict be resolved with CSS instead? Doesn’t sound like JS should be needed

Reply:
@kidonng Can you try to help me find a CSS solution? (Revert the last commit to see the issue)

Reply:
See https://github.com/sindresorhus/refined-github/pull/3296#discussion_r448624228 for a CSS-only solution.